ring-core/src/ring/middleware/cookies.clj

```diff
@@ -1,10 +1,13 @@
 (ns ring.middleware.cookies
   "Middleware for parsing and generating cookies."
-  (:import [org.joda.time DateTime Interval])
+  (:import [java.time Duration
```

Comment:
I think we can put all the `java.time` classes on the same line without loss of clarity.